aws/table_aws_efs_file_system.go

```diff
@@ -0,0 +1,256 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/efs"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+// TABLE DEFINITION
+
+func tableAwsElasticFileSystem(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_efs_file_system",
+		Description: "AWS Elastic File System",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("file_system_id"),
+			ShouldIgnoreError: isNotFoundError([]string{"FileSystemNotFound"}),
```

Comment:
Handle validation error. It should return error for these cases
![image](https://user-images.githubusercontent.com/38218418/110777365-0de15880-8287-11eb-8d3c-54128b199284.png)